core/App/localization/fr/index.ts

```diff
@@ -36,6 +36,20 @@ const translation = {
         "Unknown": "Erreur inconnue",
         "Problem": "Un problème est survenu",
         "ErrorCode": "Code d'erreur",
+        "Title1025": "Unable to reject",
+        "Message1025": "There was a problem while rejecting the",
```

Comment:
is this sentence complete?

Reply:
Probably does not need the word `while` in English.

Reply:
We would recommend having 2 unique sentences due to the french grammatical context. It would be much more intuitive.

Reply:
Ok. I'll add two to the English version and update the code accordingly.